Resources/Prototypes/_Sunrise/Entities/Objects/Fun/toys.yml

```
@@ -885,3 +885,47 @@
         volume: 5
   - type: UseDelay
     delay: 5
+
+- type: entity
+  id: PlushieMiraKovalskaya
+  parent: BasePlushie
+  name: Mira Kovalskaya
+  description: She's like to say "Meow" and "mrrr"?.. Just like francus
```

```suggestion
  description: She's like to say "Meow" and "mrrr"?.. Just like francus
```

Исправь описание в прототипе что бы скрипты нормально актуализировали перевод